Mathlib/NumberTheory/Cyclotomic/Rat.lean

```diff
@@ -344,6 +344,91 @@ theorem subOneIntegralPowerBasis'_gen_prime [IsCyclotomicExtension {p} ℚ K]
     (hζ : IsPrimitiveRoot ζ ↑p) :
     Prime hζ.subOneIntegralPowerBasis'.gen := by simpa using hζ.zeta_sub_one_prime'
 
+open nonZeroDivisors in
+/-- The norm, relative to `ℤ`, of `ζ ^ p ^ s - 1` in a `p ^ (k + 1)`-th cyclotomic extension of `ℚ`
+is p ^ p ^ s` if `s ≤ k` and `p ^ (k - s + 1) ≠ 2`. -/
+lemma toInteger_pow_sub_one_norm_prime_pow_ne_two [IsCyclotomicExtension {p ^ (k + 1)} ℚ K]
```

Comment:
I think the name should be more like `norm_toInteger_pow_sub_one_of_prime_pow_ne_two` (and similarly for the others and the already existing ones).

Reply:
Done, thanks.